plutus-ir/plutus-ir.cabal

```diff
@@ -72,6 +72,7 @@ library
         mtl -any,
         mmorph -any,
         prettyprinter -any,
+        recursion-schemes -any,
```

Comment:
Is this still needed now?

Reply:
never mind, I found it!

Reply:
Yeah, it won't be needed if I move the traversals for `Type` to `language-plutus-core`, but I thought I'd see how it goes here before exporting the approach!